editor/connections_dialog.cpp

```diff
@@ -899,9 +899,15 @@ Control *ConnectionsDockTree::make_custom_tooltip(const String &p_text) const {
 		return nullptr;
 	}
 
-	EditorHelpBit *help_bit = memnew(EditorHelpBit(p_text));
-	EditorHelpBitTooltip::show_tooltip(help_bit, const_cast<ConnectionsDockTree *>(this));
-	return memnew(Control); // Make the standard tooltip invisible.
+	// Show the custom tooltip only if it is not already visible.
+	// The Viewport will retrigger make_custom_tooltip every few seconds
+	// because the return control is not visible even if the custom tooltip is displayed.
+	if (!EditorHelpBitTooltip::is_tooltip_visible()) {
+		EditorHelpBit *help_bit = memnew(EditorHelpBit(p_text));
+		EditorHelpBitTooltip::show_tooltip(help_bit, const_cast<ConnectionsDockTree *>(this));
+	}
```

Comment:
I think we can put this logic in `show_tooltip()` instead of duplicating it in multiple places.

Reply:
Done, see my other comment!